app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -31,13 +31,10 @@ import androidx.compose.material.Divider
 import androidx.compose.material.DropdownMenu
 import androidx.compose.material.DropdownMenuItem
 import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
+import androidx.compose.runtime.*
```

Comment:
nit: Don't use * imports. Also, check if you are running the project's style configurations https://github.com/mozilla-mobile/android-components#style.